docs/sources/auth/azuread.md

```diff
@@ -0,0 +1,103 @@
++++
+title = "Azure AD OAuth2 Authentication"
+description = "Grafana OAuthentication Guide "
+keywords = ["grafana", "configuration", "documentation", "oauth"]
+type = "docs"
+[menu.docs]
+name = "Azure AD"
+identifier = "azuread_oauth2"
+parent = "authentication"
+weight = 3
++++
+
+# Azure AD OAuth2 Authentication
+
+To enable the Azure AD OAuth2 you must register your application with Azure AD. Azure AD will generate a client ID and secret key for you to use.
+
+## Create Azure AD application
```

Comment:
Per the Documentation Style Guide, avoid having only one child heading. There should be at least two h2 headings under an h1, or none.

Reply:
I changed it to a h1, but I am not sure if that was correct.

Reply:
I think the best way go forward here is to use H1 for the header and then split the article into two parts. One for the guide of setting up the application on the azure side and one for enabling oauth in grafana. We should use H2 for those headers. If you want you can change that, or we can fix it after merging.

Reply:
Ok, it looks like a minor thing, so we can merge it now and then update docs styles.